src/utils/notify_plugins/notify_email.py

```diff
@@ -1,9 +1,12 @@
+from collections import Iterable
+
 from django.core.mail import EmailMultiAlternatives
 from django.utils.html import strip_tags
 
 from utils import setting_handler
 from utils import notify
 
+DUMMY_EMAIL_DOMAIN = "@journal.com"
```

Comment:
Do you think we should set this in settings.py so it can be altered?

Reply:
Alright